### PR TITLE
Add battlepass packets

### PR DESCRIPTION
**Pull request by Erwann9875**

### CODE REVIEW

File: src/NosCore.Packets/ClientPackets/Player/BpmPacket.cs [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+        public int MaxBattlePassPoints { get; set; }
+
+        [PacketIndex(3)]
+        public int Unknow2 { get; set; } = 21093011; // todo : to find
+
+        [PacketIndex(4)]
+        public int Unknow3 { get; set; } = 21111111; // todo : to find
+
+        [PacketIndex(5)]
+        public List<BpmSubTypePacket>? QuestList { get; set; }
```
Comment (erwan-joly) [inline src/NosCore.Packets/ClientPackets/Player/BpmPacket.cs:32]: can this be null ?

File: src/NosCore.Packets/ClientPackets/Player/BpmSubTypePacket.cs [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+// -----------------------------------
+
+using NosCore.Packets.Attributes;
+
+namespace NosCore.Packets.ClientPackets.Player
+{
+    public  class BpmSubTypePacket : PacketBase
+    {
+        [PacketIndex(0)]
+        public short MissionSubType { get; set; } // todo : to enum
```
Comment (erwan-joly) [inline src/NosCore.Packets/ClientPackets/Player/BpmSubTypePacket.cs]: enum please

File: src/NosCore.Packets/ClientPackets/Player/BpmSubTypePacket.cs [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+        [PacketIndex(1)]
+        public short MissionType { get; set; } // todo : to enum
```
Comment (erwan-joly) [inline src/NosCore.Packets/ClientPackets/Player/BpmSubTypePacket.cs]: enum please

File: src/NosCore.Packets/NosCore.Packets.csproj [hunk 4]
```diff
@@ -12,7 +12,7 @@
 		<RepositoryUrl>https://github.com/NosCoreIO/NosCore.Packets.git</RepositoryUrl>
 		<PackageIconUrl></PackageIconUrl>
 		<PackageTags>nostale, noscore, chickenapi, nostale private server source, nostale emulator</PackageTags>
-		<Version>2.0.0</Version>
+		<Version>2.0.1</Version>
```
Comment (erwan-joly) [inline src/NosCore.Packets/NosCore.Packets.csproj]: minor

File: src/NosCore.Packets/ClientPackets/Player/BpmPacket.cs (hunk 1 above)
Comment (Erwann9875) [reply src/NosCore.Packets/ClientPackets/Player/BpmPacket.cs:32]: Yeah, on official if it's null there, no missions are showed up

File: src/NosCore.Packets/Enumerations/BattlePassItemType.cs [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+// |_|\__|\__/ |___/ \__/\__/|_|_\___|
+// -----------------------------------
+
+namespace NosCore.Packets.Enumerations
+{
+    public enum BattlePassItemType : byte
+    {
+        Free = 0,
+        Premium = 1,
+        GetAll = 2
```
Comment (erwan-joly) [inline src/NosCore.Packets/Enumerations/BattlePassItemType.cs]: Just All

File: src/NosCore.Packets/Enumerations/TaskType.cs [hunk 6]
```diff
@@ -0,0 +1,9 @@
+﻿namespace NosCore.Packets.Enumerations
+{
+    public enum TaskType : byte
```
Comment (erwan-joly) [inline src/NosCore.Packets/Enumerations/TaskType.cs]: FrequencyType

File: src/NosCore.Packets/NosCore.Packets.csproj [hunk 7]
```diff
@@ -12,7 +12,7 @@
 		<RepositoryUrl>https://github.com/NosCoreIO/NosCore.Packets.git</RepositoryUrl>
 		<PackageIconUrl></PackageIconUrl>
 		<PackageTags>nostale, noscore, chickenapi, nostale private server source, nostale emulator</PackageTags>
-		<Version>2.0.0</Version>
```
Comment (erwan-joly) [inline src/NosCore.Packets/NosCore.Packets.csproj]: Version need to be updated

File: src/NosCore.Packets/NosCore.Packets.csproj [hunk 8]
```diff
@@ -12,7 +12,7 @@
...
+		<Version>7.0.0</Version>
```
Comment (erwan-joly) [inline src/NosCore.Packets/NosCore.Packets.csproj]: Minor change

File: src/NosCore.Packets/Enumerations/BattlePassItemType.cs [hunk 9]
```diff
@@ -0,0 +1,15 @@
...
+// | | ' | \/ |`._`.| \_| \/ | v / _|
+// |_|\__|\__/ |___/ \__/\__/|_|_\___|
+// -----------------------------------
+
+namespace NosCore.Packets.Enumerations
+{
+    public enum BattlePassItemType : byte
+    {
+        Free = 0,
+        Premmium = 1,
```
Comment (erwan-joly) [inline src/NosCore.Packets/Enumerations/BattlePassItemType.cs]: Premium

**Review by erwan-joly — CHANGES_REQUESTED** [review]

**Pull request merged**